actions/class.Items.php

```diff
@@ -153,6 +153,15 @@ public function getOntologyData()
         return parent::getOntologyData();
     }
 
+    /**
+     * view class label action
+     * @requiresRight classUri READ
+     */
+    public function viewClassLabel()
```

Comment:
Can we declare here a return type?

Reply:
parent method doesn't has a return type, so I think this one shouldn't either, or we can not return at all, but all other methods are written in the same way